packages/material-ui-icons/package.json

```
@@ -3,7 +3,7 @@
   "version": "5.0.0-alpha.22",
   "private": false,
   "author": "Material-UI Team",
-  "description": "Material Design Svg Icons converted to Material-UI React components.",
+  "description": "Material Design SVG Icons compatible with Material-UI.",
```

No mention of React?

---

```suggestion
  "description": "Material Design icons exposed as SVG React components.",
```

---

```suggestion
  "description": "Material Design icons distributed as SVG React components.",
```
? (Anything other than "exposed" 😀 (